### PR TITLE
Created the Gitlab CI.

### PR DESCRIPTION
**Pull request by vklohiya**
Added the GitLab CI with and without glider provisioning.

### CODE REVIEW

**Comment by ssurenr** [comment]
@vklohiya Let this PR come to master

File: build-tools/attributions-generator.sh [hunk 1]
```diff
@@ -10,10 +10,15 @@ set -x
 ATTR_GEN_IMG=f5networksdevel/attributions-generator:latest
 docker pull ${ATTR_GEN_IMG}
 
+LOCAL_USER_ID=$(id -u)
+if [ "$GITLAB_CI" == true ]; then
+  LOCAL_USER_ID=9001
```
Comment (ssurenr) [inline build-tools/attributions-generator.sh]: Why are we hardcoding this?

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        yum install git unzip wget make openssh python python-pip  -y
+        pip install ansible==2.6.5
+        # Cloning Glider Repo
+        export GIT_ASKPASS=${PWD}/git_config.sh
+        mkdir glider
+        echo $GLIDER_REPO
+        pushd glider
+        echo -e '#!/bin/bash \necho $GIT_PASSWORD' > $GIT_ASKPASS
+        chmod +x $GIT_ASKPASS
+        git init && git config http.sslVerify false
```
Comment (ssurenr) [inline .gitlab-ci.yml]: Let's either use proper CA certificates to trust or use SSH Keys for authentication.

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+        git config credential.https://gitswarm.f5net.com.username ${GITLAB_USER_LOGIN}
```
Comment (ssurenr) [inline .gitlab-ci.yml]: `gitswarm` is deprecated. Let's use `gitlab.f5net.com`

File: build-tools/attributions-generator.sh (hunk 1 above)
Comment (vklohiya) [reply build-tools/attributions-generator.sh]: As GITLAB container does not have any other user created we are using the root user, however we need some id for CIS container to run, hence using 9001, earlier it was using the ID of user which is running the make prod command.

**Comment by ssurenr** [comment]
Tracked in #980

**Pull request closed without merging**